app/src/main/res/values-am/strings.xml

```diff
@@ -1,15 +1,11 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <resources>
-  <string name="menu_help">እርዳታ</string>
-  <string name="menu_home">መኖርያ ገጽ</string>
-  <string name="menu_forward">ወደ ፊት ለመመልስ</string>
-  <string name="menu_back">ወደ ኋላ ለመመልስ</string>
-  <string name="menu_settings">ምርጫዎች</string>
-  <string name="menu_search">ፍለጋ</string>
-  <string name="menu_randomarticle">መጣጥፍ በነሲብ</string>
-  <string name="menu_fullscreen">ሙሉ መጋረጃ</string>
-  <string name="search_label">ፍለጋ</string>
-  <string-array name="pref_zoom_entries">
-  </string-array>
-  <string name="pref_language_title">ቋንቋ</string>
```

Comment:
Indentation issues.

Reply:
Can you please check again because those lines have been removed, not added. However, even I check all the files manually again and can't locate any other identation issues.

Reply:
Literally just below those lines. There are lots of examples in the files changed section...